src/gameClasses/components/MobileControlsComponent.js

```diff
@@ -126,19 +113,6 @@ var MobileControlsComponent = IgeEntity.extend({
 	configure: function (abilities) {
```

Comment:
this stays as is because abilities come from game data and define which controls should be displayed on the screen on mobile. this is unrelated to rendering, rather related to taro mobile controls configuration logic